controllers/users.js

```diff
@@ -22,14 +22,14 @@ router.get('/:id/spots', function(req, res) {
 // Create a new user
 router.post('/', function(req, res) {
   if (!req.body.hasOwnProperty('email') ||
-      !req.body.hasOwnProperty('password') ||
+      !req.body.hasOwnProperty('uid') ||
         !req.body.hasOwnProperty('firstName') ||
           !req.body.hasOwnProperty('lastName')) {
     res.status(400).json({error: "Missing expected body parameter."});
     return;
   }
 
-  User.create(req.body.email, req.body.password, req.body.firstName,
+  User.create(req.body.email, req.body.uid, req.body.firstName,
```

Comment:
Yeah you can get an email from the facebook auth (and we do) , but I feel like it's still a good idea to store it separately in case the user wants to use a different email or something